### PR TITLE
BUGFIX: ONE-4507 Fix the path to the columnWidths in properties

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**
Reflect final structure of properties. Use getProperties helper everywhere

---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/3172
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2890

# Related Jira tasks
- ONE-4507: https://jira.intgdc.com/browse/ONE-4507

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7746/ : SUCCESS in 3m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3878/ : SUCCESS in 4s

**Comment by ivan-nejezchleb** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1656.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3881/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3401/ : SUCCESS in 19m 53s

**Comment by ivan-nejezchleb** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4038/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3882/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3402/ : SUCCESS in 12m 22s

File: src/components/uri/Visualization.tsx [hunk 1]
```diff
@@ -173,9 +173,7 @@ function fetchLocalizationFromUser(sdk: SDK, projectId: string): Promise<Localiz
 }
 
 function getWidthDefs(mdObjectContentProperties: IVisualizationProperties): ColumnWidthItem[] | undefined {
-    return mdObjectContentProperties && mdObjectContentProperties.widthDefs
-        ? mdObjectContentProperties.widthDefs
-        : undefined;
+    return get(mdObjectContentProperties, ["controls", "columnWidths"], undefined);
```
Comment (patrikbraborec) [inline src/components/uri/Visualization.tsx:176]: `undefined` is not necessary. Lodash `get` returns by default `undefined` (https://lodash.com/docs/4.17.15#get).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7751/ : SUCCESS in 7m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3885/ : SUCCESS in 15s

**Comment by patrikbraborec** [comment]
LGTM

File: src/components/uri/Visualization.tsx (hunk 1 above)
Comment (ivan-nejezchleb) [reply src/components/uri/Visualization.tsx]: fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7752/ : SUCCESS in 7m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3886/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4039/ : SUCCESS in 10m 28s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/94/ : SUCCESS in 5m 16s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7753/ : SUCCESS in 3m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1869/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2785/ : SUCCESS in 18m 02s

**Pull request merged**